doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable storage. |
+| `req.inf.ntw.01` | Network | The Architecture **must** support intra-pod communications such as between all containers within a pod on localhost |
+| `req.inf.ntw.02` | Network | The Architecture **must** support intra-node communications, such as between agents on a node and all pods on that node |
+| `req.inf.ntw.03` | Network | The Architecture **must** support inter-node communications without NAT, such as communications between pods on a node with all other pods on all nodes |
+| `req.inf.ntw.04` | Network | The Architecture **must** support low latency and high throughput traffic needs. |
+| `req.inf.ntw.05` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.06` | Network | The Architecture **must** support network resiliency. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems. |
+| `req.inf.ntw.06` | Network | The Architecture **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.09` | Network | The Architecture **must** be capable of supporting highly available (Five 9’s or better) CNF/VNF workloads. |
+| `req.inf.ntw.10` | Network | The Architecture **should** be architected to provide a standardised, scalable, and repeatable deployment model across all applicable NFVI sites. |
+| `req.inf.ntw.11` | Network | The SDN solution **should** be configurable in an automated manner using openly published API definitions. |
+| `req.inf.ntw.12` | Network | The SDN solution **should** be able to be centrally administrated and configured. |
+| `req.inf.ntw.13` | Network | The Architecture **must** support dual stack IPv4 and IPv6 in Pods and Services. |
+| `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
+| `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
+
+<p align="center"><b>Table 2-2:</b> Containerised Platform: Infrastructure Requirements.</p>
+
+<a name="2.3.3"></a>
+### 2.3.3 Containerised Infrastructure Management (CIM)
+
+| Ref # | sub-category | Description |
+|----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `req.cim.01` | General | The Architecture **must** allow infrastructure resource sharing. |
+| `req.cim.02` | General | The Architecture **must** allow discovery and management of NFVI resources. |
+| `req.cim.03` | General | The Architecture **must** support Enhanced Platform Awareness (EPA). |
+| `req.cim.04` | General | The Architecture **must** include image repository management. |
```

Comment:
could we generalize this to "artifact management"? besides image we might need to care about Helm charts, or any other deployment artifacts required to deploy a VNF

Reply:
+1 - I'll add both capabilities to the diagram in #449.

Reply:
However, I think the word "management" is not needed in this requirement - we must include image repository capabilities, and / or Kubernetes artefact repository capabilities, but the way it's worded suggests the management of that capability (i.e. LCM of a container registry) rather than providing a container registry.

Reply:
Changed to: "The Architecture **must** include kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities."